src/main/java/org/jenkinsci/plugins/workflow/cps/CpsFlowExecution.java

```diff
@@ -1305,24 +1305,24 @@ public void pause(final boolean v) throws IOException {
     }
 
     @Restricted(DoNotUse.class)
-    @Terminator public static void suspendAll() throws Exception {
-        ACL.impersonate(ACL.SYSTEM, new NotReallyRoleSensitiveCallable<Void,Exception>() { // TODO Jenkins 2.1+ remove JENKINS-34281 workaround
-            @Override public Void call() throws Exception {
-                LOGGER.fine("starting to suspend all executions");
-                for (FlowExecution execution : FlowExecutionList.get()) {
```

Comment:
[diff for reviewers](https://github.com/jenkinsci/workflow-cps-plugin/pull/144/files?w=1#diff-3eec43b20e92e7596aa70d1c7dacc1c6L1308)